### PR TITLE
Disable terminator

### PR DESCRIPTION
**Pull request by MSWCRB**
PR created per the issue [here](https://github.com/ParticularLabs/NServiceBus.FileBasedRouting/issues/32)

### CODE REVIEW

**Comment by timbussmann** [comment]
Hey @MSWCRB 
Thanks for your PR, we will have a review of the suggested changes soon.

File: src/NServiceBus.FileBasedRouting.sln.DotSettings [hunk 1]
```diff
@@ -591,14 +592,20 @@ II.2.12 &lt;HandlesEvent /&gt;&#xD;
...
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateThisQualifierSettings/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsCodeFormatterSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsParsFormattingSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsWrapperSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EXml_002ECodeStyle_002EFormatSettingsUpgrade_002EXmlMoveToCommonFormatterSettingsUpgrade/@EntryIndexedValue">True</s:Boolean>
```
Comment (timbussmann) [inline src/NServiceBus.FileBasedRouting.sln.DotSettings]: changes to this file need to be reverted.

**Review by timbussmann — COMMENTED** [review]
Changes look good to me. I'm wondering whether that option should be named just `EnableMessageDrivenSubscriptions` instead?

**Comment by MSWCRB** [comment]
Hey Tim – I’m fine with that name – whatever makes sense to you guys.  In addition to the reversion you mentioned, I also need to update the overloaded methods in FileBasedRoutingConfigExtensions.  I wanted to make sure all was good before I went forward with the rest of the changes there.

Thanks!

**Comment by SzymonPobiega** [comment]
@MSWCRB let us know when you update these methods. If you are busy, we can take it over from here and add them. Whatever works best for you.

**Comment by MSWCRB** [comment]
@SzymonPobiega I did already.  I haven't pushed since I was waiting for your decision on the option naming...

**Comment by SzymonPobiega** [comment]
@MSWCRB Let's go with what @timbussmann suggested, `EnableMessageDrivenSubscriptions`

**Comment by MSWCRB** [comment]
@SzymonPobiega PR created

**Review by SzymonPobiega — APPROVED** [review]

**Comment by SzymonPobiega** [comment]
LGTM. @timbussmann ?

**Review by timbussmann — APPROVED** [review]

**Comment by timbussmann** [comment]
Thanks for the PR @MSWCRB 👍 These changes are now merged into develop which means they are automatically released as part of the next minor version (2.1) which is also targeting NServiceBus 7. Are you planning to use NSB 7 soon?

**Comment by MSWCRB** [comment]
@timbussmann YW.  We are using v7 for a project we are currently developing.  We are still 1+ month away from a deployment.

**Comment by timbussmann** [comment]
that's good to hear. Once v7 is officially released we can push out 2.1 of FBR quickly.

**Pull request merged**